concepts/basics/introduction.md

```diff
@@ -0,0 +1,83 @@
+# Introduction
+
+To start the C++ journey we are starting with variables, function calls, and comments.
+
+## Variables
+
+C++ is a typed language.
+The type of a variable has to be stated upon the declaration of a variable.
+Its type cannot change during its lifetime.
+An integer variable with the name `years` can be declared like this:
+
+```cpp
+int years;
+```
+
+It is good practice to initialize variables upon declaration.
+C++ offers different mechanisms to do so.
+The version with the curly braces is more in line with modern C++, but the equal-sign version is also very common.
+
+```cpp
+int tomatoes{80};
+int potatoes = 40;
+```
+
+## Arithmetic Operations
+
+Arithmetic expressions like `*`, `+`, or `-` can be part of a statement. 
+
+```cpp
+3 * 2;
+// => 6
+```
+
+## Updating Variables
+
+You can reassign variables, as long as keep their type:
+
+```cpp
+tomatoes = tomatoes - 5; // tomatoes is now 75
+potatoes = (32 * 2) + 11; // potatoes is now 75 as well
+```
+
+## Functions
```

Comment:
It doesn't seem like a lot to me to be fair, it's really a list of essential things to know about C++ functions to be able to write them. I wouldn't add anything else on it though.

Reply:
I would rather stick to "Just give the user enough to solve the current exercise." Do they need to know what a statement is or that `return` is a special statement or how evaluations work? Because there are many exceptions to "easy rules" in C++.